CUDADataFormats/Track/test/TrajectoryStateSOA_t.h

```diff
@@ -0,0 +1,75 @@
+#include "CUDADataFormats/Track/interface/TrajectoryStateSoA.h"
+
+using Vector5d = Eigen::Matrix<double, 5, 1>;
+using Matrix5d = Eigen::Matrix<double, 5, 5>;
+
+__host__ __device__ Matrix5d loadCov(Vector5d const& e) {
+  Matrix5d cov;
+  for (int i = 0; i < 5; ++i)
+    cov(i, i) = e(i) * e(i);
+  for (int i = 0; i < 5; ++i) {
+    for (int j = 0; j < i; ++j) {
+      double v = 0.3 * std::sqrt(cov(i, i) * cov(j, j));  // this makes the matrix pos defined
```

Comment:
could a comment be added in order to make somehow more clear the meaning of these `0.3`, `-0.4` and `0.1` ?

Reply:
It's a trick copied from root (to make matrix pos defined as the comment explains)

Reply:
and exists in some legacy test as well